### DISCUSSION TITLE
VPN/代理 讨论区

### DISCUSSION THREAD

**Discussion by hugetiny** [hugetiny/awesome-vpn#38] [Announcements]
# #👋 Welcome!  

 Find truly free VPNs together, feel free to say ✏️ 

# #👋欢迎!  

一起找到真正免费的VPN吧，大家可以畅所欲言✏️

**Comment by zhq195** [comment]
咋没人啊

**Comment by 1436280166** [comment]
May I ask how to handle the configuration file?

**Comment by 1436280166** [comment]
Entering the subscription link is useless, I don't know why

**Comment by hugetiny** [comment]
把订阅链接放在浏览器中看是否能打开，如果能打开，再尝试订阅链接添加，你是什么客户端？

**Comment by 1436280166** [comment]
clash，outline都用了

**Comment by hugetiny** [comment]
clash的订阅文件格式是不同的选一个V2RAY或者XRAY端的就支持

**Reply by hugetiny** [reply]
> clash的订阅文件格式是不同的选一个V2RAY或者XRAY端的就支持

现在支持clash系和singbox系的所有配置文件格式

**Comment by 1436280166** [comment]
有ios的吗

**Comment by LHNB6666** [comment]
进入v2rayN界面好迷茫，不知道该干什么。没有导入按钮，能指点一下吗
